kotest-framework/kotest-framework-multiplatform-plugin-gradle/build.gradle.kts

```diff
@@ -90,3 +73,39 @@ tasks {
       }
    }
 }
+
+
+val updateKotestPluginConstants by tasks.registering {
+   val kotestConstantsFileContent: String = """
+            |// Do not edit manually. This file was created by ${this.path}
+            |
+            |package io.kotest.framework.multiplatform.gradle
+            |
+            |const val KOTEST_COMPILER_PLUGIN_VERSION: String = "${Ci.gradleVersion}"
+            |
+         """.trimMargin()
+   inputs.property("kotestConstantsFileContent", kotestConstantsFileContent)
+
+   val kotestConstantsOutputFile = project.layout.projectDirectory.file(
+      "src/main/kotlin/io/kotest/framework/multiplatform/gradle/kotestPluginConstants.kt"
+   )
+   outputs.file(kotestConstantsOutputFile)
+
+   doLast {
+      logger.lifecycle("Updating Kotest Gradle plugin constants\n${kotestConstantsFileContent.indexOf("  > ")}\n")
```

Comment:
Is this logging still required?

Reply:
I think some logging is useful. It will only run when the input/output changes, so it make it obvious when something's happened, and logging the full file makes it easy to search the whole project to find what's going on.

Reply:
I just realised I put `indexOf` instead of `prependIndex` - it's now fixed

Reply:
Okay yeah the logging got old quickly (plus it happens more regularly now the file isn't checked into source). I changed it to be a debug.